src/components/MainSidebar/DimensionItem/DimensionItem.js

```diff
@@ -1,11 +1,31 @@
+import { IconAdd16 } from '@dhis2/ui'
 import { useSortable } from '@dnd-kit/sortable'
 import { CSS } from '@dnd-kit/utilities'
 import PropTypes from 'prop-types'
 import React from 'react'
 import { useDispatch } from 'react-redux'
-import { acSetUiOpenDimensionModal } from '../../../actions/ui.js'
-import DimensionMenu from '../../DimensionMenu/DimensionMenu.js'
+import {
+    acAddUiLayoutDimensions,
+    acRemoveUiLayoutDimensions,
+    acSetUiOpenDimensionModal,
+} from '../../../actions/ui.js'
 import { DimensionItemBase } from './DimensionItemBase.js'
+import { DimensionItemButton } from './DimensionItemButton.js'
+
+// Joe prefers this icon as the one in ui is too light
```

Comment:
You can pass the `color` prop to the icon like you did for `IconAdd16` or does that not work for this one?